src/styles/blocks/price.scss

```
@@ -0,0 +1,18 @@
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin: 24px 0 16px;
```

move `margin-bottom: 24px` to `.card__rating`
```suggestion
  margin-bottom: 16px;
```